sdk/core/azure-core/azure/core/pipeline/policies/distributed_tracing.py

```diff
@@ -60,52 +59,50 @@ def set_header(self, request, span):  # pylint: disable=no-self-use
         Sets the header information on the span.
         """
         headers = span.to_header()
-        request.http_request.headers.update(headers)  # type: ignore
+        request.http_request.headers.update(headers)
 
     def on_request(self, request):
         # type: (PipelineRequest) -> None
-        parent_span = tracing_context.current_span.get()
-        wrapper_class = settings.tracing_implementation()
-        original_context = [parent_span, None]
-        if parent_span is None and wrapper_class is not None:
-            current_span_instance = wrapper_class.get_current_span()
-            original_context[1] = current_span_instance
-            parent_span = wrapper_class(current_span_instance)
-
-        if parent_span is None:
+        span_impl_type = settings.tracing_implementation()
+        if span_impl_type is None:
             return
 
-        path = urllib.parse.urlparse(request.http_request.url).path  # type: ignore
+        path = urllib.parse.urlparse(request.http_request.url).path
```

Comment:
Can this throw? Which brings the question "what should we do if the distributed tracing policy throws an exception"? It would be unfortunate if this is the first spot where an incorrectly formatted request URL was encountered - but only if tracing was enabled.

I would assume that we never want to throw from this method, but rather log a warning if something went bad.

Reply:
make sense, make the tracing as low disturbing as possible

Reply:
fixed